WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -801,13 +836,12 @@ import WordPressComAnalytics
     // MARK: - Sync Methods
 
     func updateLastSyncedForTopic(objectID:NSManagedObjectID) {
-        do {
-            let topic = try managedObjectContext().existingObjectWithID(objectID) as! ReaderAbstractTopic
-            topic.lastSynced = NSDate()
-            ContextManager.sharedInstance().saveContext(managedObjectContext())
-        } catch let error as NSError {
-            DDLogSwift.logError("Failed to update topic last synced date: \(error.localizedDescription)")
+        guard let topic = (try? managedObjectContext().existingObjectWithID(objectID)) as? ReaderAbstractTopic else {
+            DDLogSwift.logError("Failed to retrive an existing topic when updating last sync date.")
```

Comment:
>  is that error information not useful?

As far as `existingObjectWithID` goes the specific error doesn't seem to useful to me as its mostly related to core data internals. I think just knowing that an expected object wasn't found is enough to debug.